sdk/scripts/iot/Generate-X509-Certificate.ps1

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env pwsh
+
+param(
+[Parameter(Mandatory = $true)] [string]$SOURCES_DIR,
```

Comment:
Generally Env variables are all caps but variable names are generally camel case so `$sourcesDir`. This makes it a little clearer where these variables are comming from.

Reply:
Ok, if that's the standard with the repo, I'm not opposed, however with the env variables injested by the samples, they are caps and underscores.